tests/integration/test_users.py

```diff
@@ -33,6 +33,13 @@ def test_user_auth(client, test_database, admin_user):
     assert responseMissingToken.json == \
             {'message': 'Missing authorization header'}
 
+def test_bad_user_auth(client, test_database):
```

Comment:
Perfect thanks! Ideally this should use the `empty_test_db`, however this can be handled in a future refactoring. Thanks for getting this test added.